test/commands/add/activity_spec.rb

```diff
@@ -51,5 +51,267 @@
     end
   end
 
+  describe "adding default location" do
```

Comment:
These tests are **_AMAZING_**. 😻

Reply:
I'm so glad you think so! 🙌 😄 

I'm not gonna lie, I went into a bit of a wormhole when thinking about all the different permutaions of adding default-location activities in the past 🤯 I must have re-written the tests at least three times haha

I **wish** it was magically possible to get bug reports on gems - it'd be so cool to get a bug report on default location functionality and then really tighten the tests to cover every edge case.